### ISSUE TITLE
javascript-eslint fails with eslint v6.1.0

### ISSUE THREAD

**Issue by bioand** [flycheck/flycheck#1607]
The following predicate returns false when running eslint version 6.1.0:

```lisp
(defun flycheck-eslint-config-exists-p ()
  "Whether there is a valid eslint config for the current buffer."
  (let* ((executable (flycheck-find-checker-executable 'javascript-eslint))
         (exitcode (and executable (call-process executable nil nil nil
                                                 "--print-config" "."))))
    (eq exitcode 0)))
```

Running the equivalent command from the command line returns non-zero:

```sh
$ eslint --print-config .

Oops! Something went wrong! :(

ESLint: 6.1.0.

The '--print-config' CLI option requires a path to a source code file rather than a directory.
See also: https://eslint.org/docs/user-guide/command-line-interface#--print-config

$ echo $?
2
```
On Ubuntu 18.04.2 LTS.

In Emacs, `M-x flycheck-verify-setup`

```
Syntax checkers for buffer server.js in js-mode:

  javascript-eslint (disabled)
    - may enable:  Automatically disabled!
    - executable:  Found at /home/and/.nvm/versions/node/v10.16.0/bin/eslint
    - config file: missing or incorrect

  javascript-jshint (disabled)
    - may enable:         Automatically disabled!
    - executable:         Not found
    - configuration file: Not found

  javascript-jscs (disabled)
    - may enable:         Automatically disabled!
    - executable:         Not found
    - configuration file: Not found

  javascript-standard (disabled)
    - may enable: Automatically disabled!
    - executable: Not found

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled
checkers.

--------------------

Flycheck version: 31
Emacs version:    25.2.2
System:           x86_64-pc-linux-gnu
Window system:    x
```

**Comment by cpitclaudel** [comment]
Duplicate of https://github.com/flycheck/flycheck/issues/1600 ?

**Comment by bioand** [comment]
Indeed, it is.  My apologies.  

I have a workaround which I won't PR since I see it's fixed on master (but not yet released).

Thanks :exclamation:
